Update rulerScale.js to work with foundry v10

BrodinRaymond opened the pull request:
Updated to work with changed Ruler methods

oestred commented:
Tested successfully on V10. Works well. I would recommend updating the official files.

oestred commented:
@BrodinRaymond Doing more testing and noticed that with Easy Ruler Scale enabled now that the ctrl+drag then space to move a token along the line does not work. The console shows the following error:

```
Uncaught (in promise) TypeError: Cannot read properties of undefined (reading 'some')
[No packages detected]
    at Ruler.moveToken (foundry.js:32422:40)
    at Object._onMeasuredRulerMovement [as onDown] (foundry.js:1662:13)
    at KeyboardManager._executeKeybind (foundry.js:2147:66)
    at KeyboardManager._processKeyboardContext (foundry.js:2186:33)
    at KeyboardManager._handleKeyboardEvent (foundry.js:2224:10)
    at foundry.js:1873:54
```
Are you able to duplicate that issue? If I disable Easy Ruler Sscale the ctrl+drag then space works again. 
I think you are so close.....

BrodinRaymond commented:
@oestred thanks for the bug testing, I actually didn't realize that was a foundry feature, so didn't thing to test it when making the edits originally. Newest commit should fix the issue.

I went through and changed the module more significantly, it now only modifies the _drawMeasuredPath method of the Ruler class, rather than the whole measure method. Also removed compatability with the libRuler module as it is depreciated as of v10 and will no longer be updated.

oestred commented:
@BrodinRaymond   Awesome. The ctrl+drag and space to move a token works now and the easy ruler text size still works as exepected.
And that is great that you cleaned it up too even. 

I know a number of people used Easy Ruler Scale before so hopefully the module repository gets updated soon with your updates, or if you are interested in taking the module over I know that Foundry admins can change the pointers from one git hub to another if Kandashi was fine with you taking it over.

SophieScott12 commented:
> @BrodinRaymond Awesome. The ctrl+drag and space to move a token works now and the easy ruler text size still works as exepected. And that is great that you cleaned it up too even.
> 
> I know a number of people used Easy Ruler Scale before so hopefully the module repository gets updated soon with your updates, or if you are interested in taking the module over I know that Foundry admins can change the pointers from one git hub to another if Kandashi was fine with you taking it over.

Hello!  I would very much like to acquire this for my own use, but cannot figure out how to get the file.  Would you tell me how to acquire the edit?  I managed to download a similarly edited file for the Squeaker mod a few days ago but have no idea how I managed it.

SophieScott12 commented:
[easy-ruler-scale.zip](https://github.com/kandashi/easy-ruler-scale/files/10812828/easy-ruler-scale.zip)


> > > @BrodinRaymond Awesome. The ctrl+drag and space to move a token works now and the easy ruler text size still works as exepected. And that is great that you cleaned it up too even.
> > > I know a number of people used Easy Ruler Scale before so hopefully the module repository gets updated soon with your updates, or if you are interested in taking the module over I know that Foundry admins can change the pointers from one git hub to another if Kandashi was fine with you taking it over.
> > 
> > 
> > Hello! I would very much like to acquire this for my own use, but cannot figure out how to get the file. Would you tell me how to acquire the edit? I managed to download a similarly edited file for the Squeaker mod a few days ago but have no idea how I managed it.
> > Edit: I think I have it sorted, thanks anyway!
> 
> Can you help me with this? I cant find the manifest URL link to copy/paste into the Install Module menu in Foundry!

There isn't a manifest URL link for this version of the mod, I ended up updating it manually and I honestly don't remember how.  I've attached a zip file:  unzip that in your modules folder and replace all the files.  That should fix it.
